### ISSUE TITLE
Enable merging translations by default

### ISSUE THREAD

**Issue by swissspidy** [swissspidy/preferred-languages#864]
Once Performant Translations is merged into core, we can enable merging by default and remove the filter, since it's overall better for UX. With PT, merging should be fast enough that it shouldn't hurt site performance.

**Comment by swissspidy** [comment]
#914 enables it by default if using WP 6.5

Not removing the filter right now in case someone is using an older version of WP.
